bootstrap/config/base/kfctl_default.yaml

```diff
@@ -20,16 +20,20 @@ spec:
   - tf-training
   components:
   - metacontroller
-  - ambassador
   - argo
   - centraldashboard
   - jupyter-web-app
   - katib
   - notebook-controller
   - pytorch-operator
+  - spartakus
```

Comment:
if we do 
`kfctl init ~/kubeflow` we'll get spartakus - is that what we want? (it's not in kfctl_default.yaml)

Reply:
otherwise
/lgtm